File: Lami/01.27/2439.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

function solution() {
  const whiteSpace = ' ';
  const star = '*';
  let answer = '';
  for (let i = 1; i <= Number(input[0]); i++) {
    answer = whiteSpace.repeat(Number(input[0]) - i) + star.repeat(i);
    console.log(answer);
  }
}

solution();

File: MJ/1.17/IO/2439.py
N = int(input())

for i in range(1,N+1):
    for _ in range(N-i):
        print(' ', end = '')
    for _ in range(i):
        print('*', end='')
    print()
    
    
# for i in range(N):
#     for _ in range(i+1):
#         print(' ', end = '')
#     for _ in range(N-i):
#         print('*', end='')
    # print()